modules/data/tools/smart_recorder/realtime_record_processor.cc

```diff
@@ -19,6 +19,7 @@
 #include <algorithm>
 #include <chrono>
 #include <csignal>
```

Comment:
csignal should be put before algorithm and has its own section.

Reply:
csignal is a c++ header, it's translated from c,  so separately place first?

Reply:
> csignal is a c++ header, it's translated from c, so separately place first?

yep.